concepts/booleans/introduction.md

```diff
@@ -0,0 +1,84 @@
+# Introduction
+
+## Booleans in Julia
+
+True or false values are represented by the `Bool` type.
+It contains only two values: `true` and `false`.
+
+```julia-repl
+julia> true
+true
+
+julia> false
+false
+
+julia> typeof(true)
+Bool
+```
+
+## Boolean logic
+
+Imagine we have the following Boolean expressions in Julia: `5 > x` and `x != 0`.
+If `x` was 3 they would both be `true`.
+We can express statements like "is x less than 5 and not equal to y?" using [Boolean operators][boolean-operators]: `!` (not), `&&` (and), `||` (or).
+
+In Julia (and many other programming languages), `&&` has a [higher precedence][operator-precedence] than `||` (in the same way that `*` is applied before `+`).
+This means that `true || false && true` evaluates to `true` because it is parsed as `(true || false) && true`.
```

Comment:
```suggestion
This means that `true || false && true` evaluates to `true` because it is parsed as `true || (false && true)`.
```

However, we haven't properly introduced the logical operators at this point so students who didn't follow the Boolean operators link will likely be confused how we evaluated that expression.